app/org/maproulette/framework/repository/ChallengeSnapshotResponsitory.scala

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
So this file should be named `ChallengeSnapshotRepository` instead of `ChallengeSnapshotResponsitory`